vendor/k8s.io/kubernetes/openshift-kube-apiserver/admission/customresourcevalidation/apiserver/validate_apiserver.go

```diff
@@ -219,10 +216,7 @@ func validateCipherSuites(fieldPath *field.Path, suites []string) field.ErrorLis
 
 	if len(suites) > 0 { // something was configured
```

Comment:
Isn't it invalid for `suites` to be empty?

Reply:
Hm, the field does not have `+required` so I was just falling back to intermediate profile ciphers in this case, but the behavior of the field being empty is really not documented. 

I guess forcing it makes sense as any automatic behavior may be counterintuitive.